src/BenchmarksDriver/Program.cs

```diff
@@ -539,17 +648,21 @@ public static int Main(string[] args)
             string path,
             string method,
             IEnumerable<string> headers,
-            double rps)
+            string dimension,
+            double value)
         {
-            Log("Writing results to SQL...");
-
             const string createCmd =
                 @"
                 IF OBJECT_ID(N'dbo.AspNetBenchmarks', N'U') IS NULL
                 BEGIN
                     CREATE TABLE [dbo].[AspNetBenchmarks](
                         [Id] [int] IDENTITY(1,1) NOT NULL PRIMARY KEY,
+                        [Excluded] [bit] NOT NULL,
```

Comment:
Should we change this column to `DEFAULT 0`, rather than inserting 0 every time?  It would only be set to `1` manually as needed to cleanup bad data.